libraries/chef_pyenv_script_helpers.rb

```diff
@@ -5,12 +5,14 @@ module Pyenv
     module ScriptHelpers
       include Chef::Mixin::ShellOut
       def root_path
-        node.run_state['root_path'] ||= {}
+        node.run_state['sous-chefs'] ||= {}
+        node.run_state['sous-chefs']['pyenv'] ||= {}
+        node.run_state['sous-chefs']['pyenv']['root_path'] ||= {}
```

Comment:
Actually, shouldn't this be in the `pyenv` namespace since we own that namespace on supermarket?

Reply:
This is actually better imo. We might own the `pyenv` but by using `sous-chefs` we reduce the risk of clashing to basically 0 while using the cache.